packages/atomic/src/components/facets/base-facet/base-facet.pcss

```diff
@@ -1,3 +1,17 @@
-.facet-button {
-  border-radius: var(--facet-button-radius, 30px);
+.value-count {
+  @apply text-on-background-variant ml-1.5;
 }
+
+.value-count::before {
+  content: '(';
+}
+
+.value-count::after {
+  content: ')';
+}
+
+.value-button {
+  @apply w-full flex items-center text-left;
+  @apply py-1;
```

Comment:
Could these rules be merged into a single `@apply`? Or is there a benefit of using two?